test/new-roman-test.js

```diff
@@ -0,0 +1,63 @@
+const assert = require('chai').assert
+const convertToNewRoman = require('../lib/new-roman.js')
+
+describe('convertToNewRoman functionality', function() {
+  context('convertToNewRoman function', function(){
+    it('it can convert a number to new Roman Numerals', function(){
```

Comment:
i suggest breaking each of these use cases into separate `it` blocks to make this more organized and readable. i love that you're testing your edge cases (0 and 3001) 👍